File: csharp/examples/PostAuthResetPassword.cs
using System;

namespace Examples.examples
{
    // Example of POST /auth/reset-password
    public class PostAuthResetPassword : BaseRequest
    {
        private string _password;
        private string _token;
        
        public PostAuthResetPassword()
        {
            _token = Environment.GetEnvironmentVariable("RESET_PASSWORD_TOKEN");
            _password = Environment.GetEnvironmentVariable("PASSWORD");
           
            Console.WriteLine($"PostAuthResetPassword: _password={_password} _token={_token}");
        }

        public override void SendRequest()
        {
            var body = Newtonsoft.Json.JsonConvert.SerializeObject(new
            {
                token = _token,
                password = _password,
                password_confirmation = _password
            });

            PostRequest("auth/reset-password", body);
        }
    }
}

File: php/examples/post_auth_reset_password.php
<?php
require_once 'base_request.php';

// Example of POST /auth/reset-password

$debug = true;
$token = $_ENV['RESET_PASSWORD_TOKEN'];
$password= $_ENV['PASSWORD'];

$request_data = array(
    'token' => $token,
    'password' => $password,
    'password_confirmation' => $password
);

$request_params = '';

$json_body = json_encode($request_data);

send_request('POST', '/auth/reset-password', $request_params, $json_body);